canonical-web-and-design/ubuntu.com
# Update Ubuntu's service terms

## Done

- archive current version (index.md) to 2019-02-11.md
- update the 'Older versions' box with the newly archived version
- create new service terms based on the [copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit#)

## QA

- Go to the website https://ubuntu-com-10514.demos.haus/legal/ubuntu-advantage-service-terms
- Make sure that the archive list has a new item "2019-02-11"
- Make sure that the content of the new service terms "index.md" matches the [copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit#) *(sorry it's a very long text)*

## Issue / Card

Fixes #10483
